backend/src/repository/gridfs_bucket_collection.py

```diff
@@ -25,9 +25,9 @@ def id_exists(self, file_id):
             file_id = ObjectId(file_id)
         return self.bucket.exists(file_id)
 
-    def save_file(self, file, filename):
+    def save_file(self, file, filename, content_type):
         """Saves the file to the database using the GridFS bucket and returns the file id"""
```

Comment:
The documentation could include that it is the MIME_TYPE from a HTTP header.

Reply:
More descriptive doc string.